src/suspended.html

```diff
@@ -52,10 +52,10 @@ <h1 data-i18n="__MSG_html_suspended_toast_not_connected__"></h1>
 		</div>
 	</div>
 	<div class="watermark">
-		The Great Suspender
+		Idle Tabs
```

Comment:
It appears other text in this file uses translations from messages.json:

**Suggestion:**
```html
<div class="watermark" data-i18n="__MSG_ext_extension_name__"></div>
```